RecipeApp.Core/Units/Cooking/Gallon.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RecipeApp.Core.Units.Cooking
+{
+    public class Gallon : BaseUnit
+    {
+        public override string Name => "gallon";
```

Comment:
Instead of hardcoding the name and creating a ton of classes, create multiple objects.

Reply:
The unit itself can be an enum if you want the OOPines.

Reply:
I used this approach to be able to use reflection and multiword units... Now I use only one `Unit`class